sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -700,20 +701,33 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
 
   test("create temporary view using") {
     val csvFile =
-      Thread.currentThread().getContextClassLoader.getResource("test-data/cars.csv").toString
-    withView("testview") {
-      sql(s"CREATE OR REPLACE TEMPORARY VIEW testview (c1 String, c2 String)  USING " +
-        "org.apache.spark.sql.execution.datasources.csv.CSVFileFormat  " +
-        s"OPTIONS (PATH '$csvFile')")
+      Thread.currentThread().getContextClassLoader.getResourceAsStream("test-data/cars.csv")
```

Comment:
Could you write the comment about this? When we read the code in the future, we can know the history. Thanks!